### DISCUSSION TITLE
fatal error: FlashAsEEPROM_SAMD.h: No such file or directory

### DISCUSSION THREAD

**Discussion by RQnet** [khoih-prog/WiFiManager_NINA_Lite#20] [General]
Hi there,

i looking for your code but it doesn´t work for me. i allways get this error: fatal error: FlashAsEEPROM_SAMD.h: No such file or directory

FlashStorage_SAMD is in .pio\libdeps\SAMD installed like in this tutorial explained [(https://forum.arduino.cc/t/light-weight-credentials-wifimanager-for-teensy-sam-due-samd-stm32-wifinina/644725)]

is there any idea what to do?

i have Adruino MKR1010 board

thanks for answers

**Comment by khoih-prog** [comment]
Sorry for your inconvenience and thanks for the information.

I'm in the process of updating many libraries, and forget about the dependency of this library (too many to take care quickly after a breaking change).

Please use the latest [**FlashStorage_SAMD** releases v1.3.2](https://github.com/khoih-prog/FlashStorage_SAMD/releases/tag/v1.3.2) which has just been published to address this issue.

Regards,

**Reply by RQnet** [reply]
thanks a lot. 

i have one more problem, i get a lot of errors with dueflashstorage like this:
.pio\libdeps\mkrwifi1010\DueFlashStorage\src\DueFlashStorage.h:25:32: error: 'IFLASH1_ADDR' was not declared in this scope
or 
.pio\libdeps\mkrwifi1010\DueFlashStorage\src\efc.h:126:25: error: 'Efc' was not declared in this scope

do you have an idea?

**Reply by khoih-prog** [reply]
MKRWiFi1010 is SAMD21, and has nothing to do with DueFlashStorage (for SAM DUE).

You have to change `lib_deps = ` in `platformio.ino` or similar file to remove the dependency.

**Reply by RQnet** [reply]
yes it works. thanks a lot!

**Reply by RQnet** [reply]
i run the SAMD_WiFiNINA Example and get this Monitor output:

Starting SAMD_WiFiNINA on SAMD MKRWIFI1010
WiFiManager_NINA_Lite v1.6.0
[WN] NoWiFi
[WN] Hostname=SAMD-Master-Controller
Flag read = 0xffffffff
No doubleResetDetected
SetFlag write = 0xd0d01234
[WN] CCSum=0xe11e,RCSum=0xffffffff
[WN] ChkCrR:CrCCsum=0xaf50,CrRCsum=0xffffffff
[WN] InitCfgFile,sz=236
[WN] SaveEEPROM,Sz=1024,DataSz=0,WCSum=0xc7d
[WN] CrCCSum=0xc30
[WN] bg: isForcedConfigPortal = false
[WN] bg:Stay forever in CP:No ConfigDat

It do not open an network? do you have a idea why it not host a SAMD-Master-Controller network? i can not detect it.

**Reply by khoih-prog** [reply]
```
[WN] NoWiFi
```

means that your MKRWiFi1010 has some issue with the WiFi hardware (bad board, bad connection, etc.) and only you can find out and fix.

Try these examples to see what's wrong with your board

1. [**AP_SimpleWebServer**](https://github.com/arduino-libraries/WiFiNINA/tree/master/examples/AP_SimpleWebServer) of WiFiNINA library
2. [**AP_SimpleWebServer**](https://github.com/khoih-prog/WiFiNINA_Generic/tree/master/examples/AP_SimpleWebServer) of WiFiNINA_Generic library.

That's all the time I can have for you and your issue as this is not the problem of any library.

If you still have problem, try to isolate and/or ask for help in Arduino Forum.

Good Luck,

**Reply by RQnet** [reply]
ok thanks, both examples working well. seems the error is coming from something different
